index.d.ts

```diff
@@ -1,5 +1,12 @@
 import { Toolbox } from "./src/toolbox";
 
+export type Offset = {
+    id: string;
+    diffX: number;
+    diffY: number;
+    diffZ?: number;
+}
+
 export type ULabelAnnotation = {
```

Comment:
Is there a distinction between the class `ULabelAnnotation` and the type `ULabelAnnotation`? does typescript require you to explicitly define the type?

Reply:
You don't need to have both, but its fine as long as they both match. And sometimes it can be handy to have both of them.